DEVELOP.rst

```diff
@@ -33,7 +39,55 @@ You can run the tests like so::
 
     $ vagrant ssh
     $ cd /vagrant
-    $ ./vendor/bin/phpunit --coverage-html ./report
+
+    # Run test suite
+    composer run test
+
+    # Run code style checks
+    composer run style
+
+    # Run code style checks
+    $ ./vendor/bin/phpcs ./src
+
+
+
+************
+Using Docker
+************
+
+Installation
+============
+
+Clone the project::
+
+    git clone git@github.com:crate/crate-dbal.git
+
+Setup environment::
+
+    composer install --prefer-source
+
+
+Running the Tests
+=================
+
+::
+
+    # Run CrateDB
+    docker run -it --rm --publish 4200:4200 --publish 5432:5432 crate/crate:nightly
+
+    # Run test suite
+    composer run test
+
+    # Run code style checks
+    composer run style
+
+    # Run code style checks
+    ./vendor/bin/phpcs ./src
+
+
+*************
+Documentation
```

Comment:
can you add a "Working on the documentation" section? https://github.com/crate/crate-docs/blob/main/DEVELOP.rst is the template, so you can copy and paste from there

Reply:
Improved the title, thanks.

Reply:
Ah, do you think I should copy in the whole section?

Reply:
Done.